INN/umbrella-sfpublicpress
# Styling for secondary featured post

## Changes

This pull request makes the following changes:

- Adds `widget-content.php` partial
- Adds `Homepage Secondary Featured` widget area
- Adds styling for Largo Recent Posts in aforementioned widget area

Desktop:
![Screen Shot 2020-01-09 at 4 13 13 PM](https://user-images.githubusercontent.com/18353636/72105249-2f4d5a00-32fb-11ea-9570-1cdddd826497.png)

Mobile:
![screencapture-sfpp-test-2020-01-09-16_13_21](https://user-images.githubusercontent.com/18353636/72105248-2f4d5a00-32fb-11ea-8aa8-cbf535275d5d.png)

## Why



For #17

## Testing/Questions

Features that this PR affects:

- Largo Recent Posts in homepage secondary featured area
- Basic Largo Recent Posts widget styling


Questions that need to be answered before merging:


Steps to test this PR:

1. Add a Largo Recent Posts widget to the Homepage Secondary Featured widget area with these settings:
 - empty title
 - 1 post
 - avoid duplicates
 - large image
 - 2 sentence excerpt
 - show byline
 - hide publish date
 - show top term
2. View on frontend on desktop and mobile and verify it looks ok
